content/en/blog/2024/prometheus-compatibility-survey/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Insights from the Prometheus Compatibility Survey
+linkTitle: Prometheus Compatibility Survey
+date: 2024-06-10
+author: '[David Ashpole](https://github.com/dashpole) (Google)'
+issue: https://github.com/open-telemetry/sig-end-user/issues/24
+sig: End-User SIG
+cSpell:ignore: Ashpole
+---
+
+[Prometheus](https://prometheus.io/) and OpenTelemetry are two of the most
+active and popular projects in the
+[CNCF observability landscape](https://landscape.cncf.io/guide#observability-and-analysis--observability).
+The two communities have been working together since the early days of
+OpenTelemetry to improve the compatibility between the two projects. The
+OpenTelemetry Prometheus SIG has been leading this effort, with the active
+participation of maintainers from both OpenTelemetry and Prometheus.
+
+At this point, there is a
+[detailed, experimental specification](/docs/specs/otel/compatibility/prometheus_and_openmetrics/)
+describing how to convert between the OpenTelemetry metrics format and the
+Prometheus metrics format. It has been used to implement Prometheus (pull)
+exporters for OpenTelemetry SDKs, OTLP export from Prometheus libraries, OTLP
+ingestion for the Prometheus server, and the OpenTelemetry Collector's
+Prometheus Receiver and Prometheus exporters.
+
+One of the most challenging areas to reconcile is that OpenTelemetry metric
+names are changed when exporting to Prometheus. Today, the OpenTelemetry
+`http.server.request.duration` metric, with unit `s`, is translated to
+`http_server_request_duration_seconds` in Prometheus. Some users are familiar
+with the Prometheus naming conventions, and appreciate the consistency this
+translation provides with existing metrics in the Prometheus ecosystem. Other
+users are confused when querying for the original OpenTelemetry name does not
+return any results.
+
+Prometheus is working on support for UTF-8 characters in metric names as part of
+its
+[2024 roadmap](https://prometheus.io/blog/2024/03/14/commitment-to-opentelemetry/#support-utf-8-metric-and-label-names),
+which potentially allows preserving dots in metric names. To better understand
+what users want their Prometheus query experience to look like, we ran a survey
+with the help of the OpenTelemetry End User SIG. Deciding on the default
+translation approach is one of the last remaining blockers for stabilizing the
+compatibility specification.
+
+The survey received 86 responses, and contained many helpful pieces of feedback.
```

Comment:
```suggestion
The survey received [91 responses](https://github.com/open-telemetry/sig-end-user/blob/main/end-user-surveys/otel-prom-interoperability/otel-prom-interoperability-survey.csv), and contained many helpful pieces of feedback.
```

See [CSV results](https://github.com/open-telemetry/sig-end-user/blob/main/end-user-surveys/otel-prom-interoperability/otel-prom-interoperability-survey.csv)

Reply:
5 of the results were either spam or duplicates.  (e.g. search the results for "visa").  I've omitted them from my analysis below

Reply:
For whomever looks at the raw data or PDF doc, it may be worth mentioning that clarification for the sake of completeness.

Reply:
added